### ISSUE TITLE
DockerBuildCmd should execute docker commands to build docker image

### ISSUE THREAD

**Issue by DineshAlapati** [networknt/light-bot#6]
DockerBuildCmd utility class relies on build.sh script to build docker image. As light-codegen does not generate this file, we should execute docker commands instead of executing build.sh script.

**Comment by stevehu** [comment]
@DineshAlapati good catch. Is it possible to have a generic build.sh? If yes, we can put it into light-codegen. It is much easy to use build.sh instead docker command line. Thanks

**Comment by DineshAlapati** [comment]
Yes we can have a generic build.sh that can be generated from light-codegen but, build.sh requires image name to publish to docker hub. User has to include this image name before running it for first time or we can let the script to take this value as a argument.

**Comment by stevehu** [comment]
The image name should be in sync with service name defined in the config.json file. The only variable would be the organization of docker hub. I will try something tonight and we can have a chat tomorrow. Thanks.
